enterprise/backend/test/metabase_enterprise/serialization/models/entity_id_test.clj

```diff
@@ -108,7 +108,14 @@
     :model/CloudMigration
     :model/Comment
     :model/CommentReaction
-    :model/Workspace})
+    ;; TODO (lbrdnk 2025/12/17): I've added rest of the workspace models here as Workspace was present. I believe
+    ;; going forward all of that will be available for export. We should revisit this later in the project.
```

Comment:
Ditto - Let's rather mark all of these as these as excluded, barring the WorkspaceTransform.